install.sh

```diff
@@ -83,11 +83,13 @@ nvm_download() {
     curl --compressed -q "$@"
   elif nvm_has "wget"; then
     # Emulate curl with wget
-    ARGS=$(echo "$*" | command sed -e 's/--progress-bar /--progress=bar /' \
-                            -e 's/-L //' \
+    ARGS=$(nvm_echo "$@" | command sed -e 's/--progress-bar /--progress=bar /' \
                             -e 's/--compressed //' \
+                            -e 's/--fail //' \
+                            -e 's/-L //' \
                             -e 's/-I /--server-response /' \
                             -e 's/-s /-q /' \
+                            -e 's/-sS /-nv /' \
```

Comment:
what are all these extra options doing?

Reply:
I copied/pasted from nvm.sh to be consistent.